code/src/components/Playlists.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+export const Playlists = (props) => {
+  
+    return (
+    <article className="album">
+      {/* ===== ALBUM COVER ===== */}
+      <div className="album-cover">
+
+      
+        <a href={props.articleDetails.external_urls.spotify} target="_blank" rel="noopener noreferrer">
+          <img className="album-image" src={props.articleDetails.images[0].url} alt="album cover" />
```

Comment:
The alt needs to have a clear name for the screen reader, for example "album name" instead of "album cover" by fetching from data.json.